multi_column_editor/__init__.py

```diff
@@ -149,30 +141,35 @@
     $("#fields").html("<table class='mceTable'>" + txt + "</table>");
     $('#fields').bind('DOMNodeInserted', makeColumns);
 }
-
 // Attach event to restructure the table after it is populated
 $('#fields').bind('DOMNodeInserted', makeColumns);
 </script>
 """
 
-def getKeyForContext(self):
+def getKeyForContext(self,field=None):
     """Get a key that takes into account the parent window type and
     the note type.
     
     This allows us to have a different key for different contexts,
     since we may want different column counts in the browser vs
     note adder, or for different note types.
     """
-    return "%s-%s-%s" % (CONF_KEY_COLUMN_COUNT,
-                     self.parentWindow.__class__.__name__,
-                     self.note.mid)
-
+    key = str(self.note.mid)
+    if getConfig(self,"same config for each window",False):
```

Comment:
I think this is doing the opposite of what you intend? The default behaviour should be to have individual settings per window (which was an intentional design) but with this it covers all windows unless you explicitly turn it off.

Reply:
Feel free to send a pull request with the fix